### PR TITLE
changelog-0.65.0

### PR DESCRIPTION
**Pull request by itai-david**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
 
+## v0.65.0 - 2022-05-16
+
+What's new:
+- API expansion to support Import from UI (#3233)
+- Airflow hook wait for DAG completion (#3231)
+- Friendlier post-setup page (#3318)
+- Show skipped hooks for failed action run (#3359)
+- Import from UI (#3352) 
+- Additional documentation for user management (#3320)
```
Comment (itaiad200) [inline CHANGELOG.md:11]: Documentation changes should not be included in the changelog as they have different release cadence.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+Bug fixes:
+- UI enable row revert action when possible (#3291)
+  - Fixes bug #3161 - Bug: revert path shows up when comparing refs
+- Force catalog ID in metastore client config to be a string (#3252)
+  - Fixes bug #2996 - Glue catalog_id parsed as int
+- Fix: login redirects to credentials page (#3319)
+- UI: fix create repo focus on name (#3342)
+  - Fixes bug #3343 - UI create repository focus
+- Fixed link to authentication page vs authorization page (#3337)
+- UI api encode URI component for path parameters (#3290)
```
Comment (johnnyaug) [inline CHANGELOG.md:22]:
```suggestion
- Import your data into lakeFS using the UI! (#3233, #3352)
- Airflow hook wait for DAG completion (#3231)
- Friendlier post-setup page (#3318)
- Show skipped hooks for failed action run (#3359)

Bug fixes:
- Fixed: Revert path shows up when comparing refs (#3291)
- Fixed: Glue catalog_id parsed as int (#3252)
- Fixed: login redirects to credentials page (#3319)
- Fixed: UI create repository not focused on name field (#3342)
- Fixed link to authentication page vs authorization page (#3337)
- Fixed: Some URI parameters were not encoded (#3290)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## v0.65.0 - 2022-05-16
+
+What's new:
+- API expansion to support Import from UI (#3233)
+- Airflow hook wait for DAG completion (#3231)
```
Comment (johnnyaug) [inline CHANGELOG.md:7]: I think this is the wrong PR ID?

**Review by itaiad200 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (johnnyaug) [inline CHANGELOG.md:11]: Note: documentation changes are not part of the release - they are published immediately when merged to main

**Review by johnnyaug — APPROVED** [review]
Approved - left comments

File: CHANGELOG.md (hunk 1 above)
Comment (itai-david) [reply CHANGELOG.md]: :+1: 
Does the `include-changelog` a mistake, or should I be aware of that and exclude manually?

Comment (itai-david) [reply CHANGELOG.md]: :+1:

Comment (johnnyaug) [reply CHANGELOG.md]: It's a mistake

File: CHANGELOG.md (hunk 3 above)
Comment (itai-david) [reply CHANGELOG.md]: Right. Thanks for noticing :)
Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (itai-david) [reply CHANGELOG.md]: Thanks for the edit :pray:

**Pull request merged**